react/features/base/flags/constants.js

```diff
@@ -1,5 +1,23 @@
 // @flow
 
+/**
+ * Flag indicating if audio mute feature should be enabled.
+ * Default: enabled (true).
+ */
+export const AUDIO_MUTE_ENABLED = 'audiomute.enabled';
```

Comment:
Using feature flags for this is an interesting approach, but I think we can do it simpler: feature flags were implemented loosely, so values can also be strings. So, a simpler approach would be to have some `toolbar.buttons` flag which has a string of comma separated values such as `audiomute,hangup,videomute` etc.

Then we can split the list and check if a button should be rendered by checking if the name is in the list. This is similar to what the web already does, except that the web reads it from interface_config.js. Since we are not doing that here, using features flags sounds like a good approach!

As for recording: a little backstory -- Apple is not happy about the fact we require a Dropbox account for recording. We tried to explain that a server is doing the recording and so on, but they wouldn't listen. So if that flag is set then we hide the button to pass Apple's review. With the approach I suggested above, the recording button would be part of the list, so it would be possible to disable on Android too, but we want to keep the existing flag for iOS because it's related to the app approval process.

Reply:
@saghul I did the modification .
Added toolbar.buttons flag to enable/disable buttons. Now whatever buttons we pass in  'toolbar.buttons' from native will get enabled.
Added DEFAULT_TOOLBAR_BUTTONS, so if  toolbar.buttons did not passed from native then this default value will work.
For recording in android 'toolbar.buttons' will work and for iOS, it will check previous flag(ios.recording.enabled) and 'toolbar.buttons' both.

Let me know if any modification is required.
Thanks